R/plot_step_ahead_model_output.R

```diff
@@ -162,6 +167,9 @@ plot_step_ahead_model_output <- function(
                           plot_target = plot_target)
 
   # Data process
+  if (log_scale) {
+    model_out_tbl[["value"]] <- log10(model_out_tbl[["value"]])
```

Comment:
Trying to think about what will happen if values are zero. Then we'd have -infinity values. maybe this is ok, and would just work out. And/or people could do some custom offset transformation to accomodate this?

Reply:
Not sure there is anything that needs to be done here - just more thinking out loud.